dist.ini

```diff
@@ -36,10 +36,7 @@ tag_format = %v
 push_to = origin master
 
 [AutoPrereqs]
-skip = ^DDG::*
-
-[Run::BeforeBuild]
-run = cpanm DDG --skip-installed --mirror http://www.cpan.org/ --mirror http://duckpan.org
+skip = ^DDG::(Spice|Goodie|Fathead|Longtail|Publisher)
```

Comment:
Might we want to add this for roles too? `^DDG::((Spice|Goodie|FatHead|LongTail)(Role)?|Publisher)`.

There also appears to be a `DDG::Util` hierarchy in Spice.